### PR TITLE
Small design pass

### PR DESCRIPTION
**Pull request by argyleink**
tried to change very little but not be blinded (added dark mode) or have to read paragraphs that are super wide (added max paragraph size).

Screenshot 2024-02-22 3 47 39 PM

### CODE REVIEW

**Comment by argyleink** [comment]
i should update the other demos

File: examples/carousel/image/index.html [hunk 1]
```diff
@@ -44,10 +45,11 @@
             width: 50px;
             height: 50px;
             border-radius: 50%;
+            border: 1px solid;
             align-self: center;
             position: absolute;
             z-index: 1;
-            transition: background-color 300ms, border-color 300ms, color 300ms;
```
Comment (flackr) [inline examples/carousel/image/index.html:50]: Doesn't removing border-color break the nice transitions from the UA border-color change when the buttons become disabled / enabled?

File: examples/carousel/image/index.html [hunk 2]
```diff
@@ -44,10 +45,11 @@
...
+            transition: background-color 300ms, color 300ms;
```
Comment (argyleink) [inline examples/carousel/image/index.html:52]:
```suggestion
            transition: background-color 300ms, border-color 300ms, color 300ms;
```
restore border-color animation

File: examples/carousel/image/index.html (hunk 1 above)
Comment (argyleink) [reply examples/carousel/image/index.html]: fixed!

**Review by flackr — APPROVED** [review]

**Pull request merged**